### DISCUSSION TITLE
2005/09/22/%E6%88%91%E7%9A%84%E4%BA%BA%E5%B7%A5%E8%8D%A8%E9%BA%BB%E7%96%B9%E7%BB%8F%E5%8E%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#48] [Announcements]
# 2005/09/22/%E6%88%91%E7%9A%84%E4%BA%BA%E5%B7%A5%E8%8D%A8%E9%BA%BB%E7%96%B9%E7%BB%8F%E5%8E%86

2005年9月22日

https://blog.qizhen.xyz/2005/09/22/%E6%88%91%E7%9A%84%E4%BA%BA%E5%B7%A5%E8%8D%A8%E9%BA%BB%E7%96%B9%E7%BB%8F%E5%8E%86

**Comment by ruanqizhen** [comment]
幸福的子弹
2010年03月13日 21:05 编辑
真不幸啊，我也患了这个，一般情况下不挠就没事，一挠就痒，而且发红，略微红肿。发现这个的时候还是因为女朋友说她有这个病，而且她更厉害，吃饭都很困难，现在两三年了，这个病一直在身上，痛苦啊。。

回复
mandy
2009年12月13日 21:31 编辑
哎呀“`俺也是人工荨麻疹的受害者！！也不知道怎么得的！我一直都是皮肤超级好，从来对什么都不过敏！小时候上医院，做皮试对我来说都是多此一举！可是，这次真是让我大吃一惊！！我仔细回忆了一下，应该跟我长期熬夜和过度减肥，劳累，免疫力下降有关系。再就是可能跟我爸往家里买了好多花有关系。但我原来对花粉也不过敏呀！原来听同学有对花粉过敏的，都觉得新鲜：“花粉也能过敏？？”于是，去了大连皮肤病医院看了，也做了过敏源测试，什么也没查出来！开了中药和西药，吃了完全没效果！到后来严重了，手心都有涨疼的感觉！！这让我很紧张，就怕病情加重！但是老这么剧痒实在让我受不了。家人说我晚上睡觉的时候是不是做噩梦了，怎么老是动弹，而且翻来翻去的！我听了都害怕！后来仔细回想了一下，我动弹的时候应该还是有轻微意识的，就是因为后背还有一些地方太痒了，于是就在床单上蹭来蹭去的。哎~~于是，就去了药房。一番讨论后，买了一个叫“马来酸氯苯那敏片”的药。一天三次，一次一片。后来去了北京，还是一直没好，还有加重的趋势。但是这药好象挺管用，吃上就不痒了。但是我心特别粗，加上工作忙，老是记不起来按时吃。总是想起来就吃，想不起来就不吃了。再不就等身体有痒的感觉了，就赶紧吃药。一吃上过会儿就好了。到最后，我要是早上忘吃了，中午就吃两片。再到后来，我就每次吃两片，也不管什么时候了，一天三次，一次2片的时候也有；一天一次，一次2片的时候也有。到现在也差不多大半年快到一年了。还是一直吃的这个药。我感觉好象别的药都不好用，就这个好用。难道是个人体质不一样，药效也不一样？家人很紧张，都怕我老吃药对身体有害。但我也确实找不到别的办法。药一吃完了就得赶紧买。包里没这药心里就不踏实！不过，最近心情还算很好，也做了一些体育锻炼。反正不知不觉地，感觉病症有所好转了。基本药吃的不是那么密了。有时候，2，3天都忘记吃了，也不痒。所以现在基本就是，有痒感觉再吃。其实，我感觉治这病，增强免疫力是绝对有必要的！生活要有规律，经常做做体育锻炼。保持好的心情也是绝对有帮助的！！这些我都深有体会！最后，我希望受此病困扰的朋友们都能早日康复！也希望我自己能尽快有所好转！博主说的对，比起那些更严重的病，这点小病根本算不了什么！只要我们保持一份好的心情，平和的心态，规律的生活，适当的体育锻炼，增强了身体的免疫力，就一定能有所好转！大家一起加油 ！

回复
阿七
2009年11月20日 19:14 编辑
“我最近还有一个发现，就是：得人工荨麻疹的都是帅哥美女。”帅哥美女身材苗条，都是五行木旺的体征——秀美，但是精神或身体好动、停不下来，容易神经衰弱、气短疲劳，肺主皮，长期木旺火旺，一定会生鼻炎、皮肤病、肺病。如果再加上工作生活没规律，就会非常严重。

回复
云飞
2009年11月18日 15:04 编辑
看到这篇文章学了好多东西哦啊，呵呵，我皮肤一直就不太好，但就这两天开始特痒，挠完后马上就红，过一会儿就好了，这两天上网也看了很多，好像的确没有什么根治的方法，难受死了！不知这病会不会死终身的啊？

回复
灵
2009年10月29日 12:35 编辑
过敏性体质跟植物神经功能紊乱有关，原因一般是生活压力大，工作紧张，周围人际关系不好等造成的。过敏症状只是植物神经功能紊乱的一种表现，还有肠易激综合症、胃食管反流症状、抑郁或焦虑、失眠、嗜睡、易怒、子午潮热、头痛头晕等。

回复
Erika
2009年06月23日 15:26 编辑
我的荨麻疹治愈经历，希望能给各位朋友带来帮助哦！ 偶然从猫眼转到这里，看到那么多的病友，让我想起我得荨麻疹的一段漫长经历，我是在几年前得的这个病，记得第一次发病也是在毫无征兆的情况下突然发生的，先是脸上出现一块一块的红疹，特别是在眼睛周围和下巴至耳根处，之后是手腕处，奇痒无比！因为大多是在脸上，所以也不敢过多的抓，怕破相哦，呵呵，当时真是难受得要命啊！几个小时后，红疹慢慢的退去，也不痒了。记得好像是因为感冒吧，之前吃过几粒头孢，所以开始还以为是药物过敏反应，但是以前没有过任何对药物过敏的情况发生呀，好在之后几天也没有再发生这种情况，渐渐的就把它忘了。 几个月后，这种情况又突然发生了，而且是一发不可收拾，只要是一吹风或者一冷就马上发病，空调房间是更不能呆了，害得同事在办公室都不敢开空调，大热天的真是惨啊！红疹的面积也越来越大，从刚开始的脸上、手上，到整个腿上，而且都是一大片一大片的突出，非常的吓人，常常吓到同事和朋友，到后来他们也习惯了，因为我每天都要发病几次，症状基本在2小时后慢慢消失，那段时间真是过得郁闷啊，难受得不得了！去过好多次医院，医生确诊是荨麻疹，也吃过大家说的盐酸西替利嗪，坚持吃了一个月，戒酒，戒海鲜，戒一切有可能导致皮肤过敏的东西，海鲜可是我的最爱啊，郁闷，吃了一段时间后渐渐的好了很多，但是停药的2天后症状又开始了，后来发现盐酸西替利嗪必须长期吃，只要是一停药就会发，但是只要是药物都会有点副作用，所以我后来只是在比较严重的时候才吃1粒；经过反复折腾，得出结论：发病时吃1粒西替利嗪，症状基本上在2小时后会消失，但是你不吃药，症状也会在这段时间内消失，所以这药根本不可能治愈荨麻疹，除非你一直吃，不能停，但是预防症状发生的效果不错，前提也是你要一直吃，呵呵，可恶的副作用啊；我也试过其它药物，忘记什么名字了，甚至是中药蒸疗，但都没什么好的效果。看我的治疗一直没有什么起色，老公开始帮我在网上查找这种病资料，研究起治疗方法了，经过他的研究，我开始使用他的偏方来进行治疗，并停止吃药；方法很简单哦，先是用丝瓜的皮在患处轻轻的擦，让它的汁水渗入皮肤，嘿嘿，红肿渐渐消去了，也不痒了，看来效果不错，几小时后再洗掉；食物上我也没再忌口，只是家里的餐桌上经常会有丝瓜炒毛豆，呵呵，一段时间下来我看到丝瓜就想吐了，但是效果真的不错哦，坚持一段时间后症状明显减轻了许多，发病的次数也越来越少了，到后来就好了；期间，我还用韭菜的汁来洗澡，用韭菜汁擦拭身体后，过一段时间再洗掉，我只洗过几次，因为那味道我受不了，呵呵！现在我已经很久没有再发病了，有1年多了，应该是彻底好了吧，而且怎么吹风也没事了，即使是坐摩托车的时候，记得以前，只要一坐摩托车脸就肿起来，吓得我都不敢坐，现在在空调房间呆多久也都没问题啦，哈哈！希望我的方法能帮到你们哦！

回复
华娟
2009年05月15日 09:12 编辑
我也是，请楼主经常更新这篇文章，我想知道最后你怎么样了。有么有更好的方法啊。我也锻炼身体的，可是隔2天就发作一次，痒的难受，就吃一粒药，总不能老吃药啊。大家有没有好办法啊。有好办法的可以发Email给我，我的邮箱是qianhuajuan@126.com

回复
abby
2009年04月10日 17:44 编辑
你的经历和我简直一模一样，只是把德州改成日本而已我也很痛苦现在

回复
Hube
2009年04月9日 15:28 编辑
有没有荨麻疹治好的案例啊，我看了之后真的好灰心啊，我也是2005年开始患病的，经历过长期吃中、西药，长期忍受剧痒，但是现在的症状跟最初发病的症状有细微的变化，还练就了很好的忍功，但是痒的时候还是会下意识的抓抓，只要一抓就一发不可收拾，从点到全身，真的惨不忍睹，欲哭无泪，怎么现在的科学对过敏就没有办法根治呢？

回复
国红
2009年01月15日 10:47 编辑
半个月前我进来过，我很幸庆，这半个月没吃过一粒药，有时候还会有一点点痒，抓了也会红，但能忍住，不像以前那样受不了了。我这半个月有参加一些运动，最主要跟之前不一样的是我戒了海鲜和辣食，虽然忌口也有些痛苦，但我认为只要摆脱了那些讨厌的药，都值得。另外，我觉得得了这病，心态一定要好，平常心对待，着急只会起反作用。我希望自己这样的状态能一直保持下去并能更好，也希望各位病友能克服这个病。

回复
国红
2008年12月31日 09:45 编辑
原来有那么多病友，我们怎么办啊？我是今年8月份开始得这个病的，估计跟我搬了住的地方有关，我已经看了绍兴，杭州，上海好几个皮肤病医院，说的都是差不多的话，现在基本上3-4天吃一颗药，本还想看看是不是还有什么偏方可治，看了大家这么多评论，我好失落，看来这个病治不好了。红一点或者有一条条的我倒觉得没什么大关系，但是痒起来的时候真的受不了，只能吃药，但这样对身体肯定不好，最重要的是我这一两年打算生宝宝，老是吃药总不行啊，我怎么当妈妈呀？

回复
tigger
2008年12月15日 16:26 编辑
想请教一下，我这皮肤敏感使这半年来才开始的。。我痒的位置会一直不停的变，痒的部位会从头皮到脚板不停变换。。脸部却很少有，就算有也是一条痕而已。。而耳朵的话，会又红又热的但是却不会痒。。但是我却不像你们所说的“一刮到就会红起来”，只是皮肤在痒的时候抓，就会浮现一道一道的。。那这算不算是“人工荨麻疹”呢？

回复
超
2008年10月27日 17:17 编辑
我都不知道怎么得的开始以为是一般的过敏也没注意没想到居然是这样能不能分享一下心得啊。。。55

回复
茂
2008年10月15日 22:51 编辑
我发现好像是2006年，那时根本就没注意，去年去看了，说是荨麻疹，吃了中药  也没好，反正就是没理它了，但今年暑假痒的厉害了，而且脸部经常有，很难看，同学都说吓人，我是不怕痒。影响形象啊！注意身体真的很重要，我可能这一年多经常熬夜看电视导致更严重的，一吃那个盐酸西替利嗪片就可以保两天不痒，那个炉甘石洗剂对我是没用的，主要的是我过敏源找不到啊，有哪位大哥大姐知道的话告诉我一声，谢谢！
 

回复
zhuhong
2008年09月9日 15:33 编辑
    　我今年去做了阑尾炎手术，才知道我青霉素过敏．后来去看医生，说我不能吃阿莫西林，还有青霉素之类的药物．好痛苦，要吃什么都要想一下能不能吃，还不能加班．一吹风就又痒又痛，很难受．

回复
zhuhong
2008年09月9日 15:29 编辑
     我也是人工荨麻疹,快两年了也没好.吃过盐酸西替利嗪,开瑞坦,湿毒清,还有皿士林之类的一些药物.中药也吃过一年左右,现在吃西替和湿毒清,很怕冷,一吹风就难受,可能就会发起来.经期会发得很厉害.它害得我换工作,心情也很差.只是不知道它什么时候会好.

回复
ＫＩＤＤ
2008年09月6日 15:57 编辑
我发现得这个病是上大学期间，用硫磺皂洗手后，手上起了很多划痕 过一会自动消失了。
因为是男孩子，当时没大注意。后来有次光膀子到隔壁寝室，同学说 你背上怎么一道一道，蚊子咬了么？
后来我注意到了，有时看到两肋的划痕，心里难免感到打颤！去医院 医生没开药说严重了吃点抗过敏的药可以了。没说忌口
换了几个城市后，症状时好时坏。我也不会去主动的抓挠。很长时间都忘记我有这病了。前几天游泳洗完澡后，有人说我背上很多抓痕。这才我又注意到了！呵呵。我游完泳有次洗澡时 看到两人有类似症状！我无意中和朋友说起，两个朋友的GF都有这种症状！看来挺常见！
至于病因，我认为我是在海边城市生活，阴冷潮湿的居住环境有关！
只有自己多锻炼身体，少去注意他，我是不会忌口！注意保暖！
 
 

回复
yingmei
2008年09月5日 14:23 编辑
我也是呢。。。去年吃了大概两个月左右的药就好了，可是今年不知怎么了怎么也不好。皮肤碰到一点就红起来，真是难过。饮食方面从来不忌口，我知道海鲜和辛辣的东西可能对我不好，要是总不能避免。。。这个星期打算再去看看医生，看有没有更好的方法。

回复
Ivy
2008年09月5日 11:20 编辑
我也是十几年了。没有用任何措施治疗。
有时候会自己拿牙签在身上划图案玩。
算是可以带来乐趣的病么？
男朋友还是没有完全接受，看见突起的红肿还是会大惊小怪，拿风油精啊啥啥。
体质一直没有好过，所以不晓得体质好的时候会不会减轻。
总之没有起伏。没有特别好或者特别不好的时候。
算是一点信息么？

回复
成婷
2008年08月16日 23:02 编辑
我也是这个病症的奴隶啊~小时2，3年级就开始了。到现在12年了，还不见好……我也吃过盐酸西替利嗪和息斯敏，但是效果不好，而且还有副作用。这个病真是痛苦，奇痒无比啊！还不能抓，真是痛不欲生~
我现在在吃荨麻疹丸和特非那定片还有维生素，不知道效果怎么样（才开始吃）如果有效果讲推荐给大家，也请大家推荐点好处方~感激不尽啊！

回复
vivian
2008年08月6日 10:44 编辑
得这个病真痛苦啊，吃药就好，不吃就发，都有药物依赖了。看了你的经历，我也要努力锻炼身体！！！

回复
Unknown
2008年07月30日 18:34 编辑
我也是这个病的患者。自从三个月以前发现了症状，现在一直处于痛苦中。平时工作生活，也经常手部，脸部产生过敏症状。
万幸的是，我的症状还不是很严重。看了很多帖子后，我决定采用中医的治疗方法，如果将来我的病情减轻了或者痊愈了，一定把处方告诉各位，希望大家能够早日病愈。

回复
雾
2008年07月13日 17:06 编辑
这几个月痒得我真的好难受,以前都是冬天才会痒的,现在都夏天了,还痒疯了.去看医生,医生就说我是过敏的,开了一些给我,吃了只好一个星期. 姐姐给我买了息斯敏,晚上吃,不痒.但姐说不能长期吃会伤肝,可能是湿热,吃湿毒清吃. 很听话,吃了两瓶.一点效果都没有. 上网一查:和网上说的症状和对照图片一看才知道自己患了人工荨麻诊.

回复
Li
2008年06月12日 10:52 编辑
我有人工性荨麻疹也快又两年了，是当初在医院皮肤科实习的时候初发的。我觉得环境和饮食对这个病的影响真的很大。当初在皮肤科的时候就觉得那里真的是很脏啊，没想到实习到中期就发现身上的皮肤很容易起红包，不能抓它，一抓就惨了，所有抓过的地方就马上起风团，又红又痒。 因为在皮肤科实习，所以就就近问了我的老师这是怎么回事， 老师一看就很肯定的回答我这个是慢性荨麻症（又叫人工荨麻疹）。老师说这个病目前没有特效药，但是对生活也不会有太大的影响。我就没去管它了。想着肯定是医院太脏了，又要值夜班所以会这样的。不待医院应该就会好的。所以实习结束毕业后我就转行了。结果离开医院后情况真的好了很多，就是会有红红的抓痕。但是不会有风团。前几天老同学聚会太开心了，大家都喝了很多酒，玩到晚上两点多，当时在喝的时候就觉得手上有点痒，但是因为太开心了，所以就没去管它。第二天早上起来，手上就发出一粒粒的了，当初还以为是真菌感染，就赶快去皮肤科看看想配点药。因为真菌感染是会传染的。结果到了医院我手上的挖痕就出现风团了，在医生知道我昨晚又喝了酒后就被他狠狠地骂了顿。嘿嘿。勒令以后不能喝酒，不能吃海鲜，肉类也只能出两种：猪肉和鸭肉，55555我最爱吃海鲜啦。
   不过搂主的美女帅哥论我是十分的赞同啦。嘿嘿

回复
J
2008年02月23日 23:57 编辑
我得这个病有23年了，5岁搬家时得的寻麻疹，到今天28岁试过无数的医院，药方没有有效的。我是一遇到冷就起红点点，还很痒。经过了20年的治疗不见气色，3年前我放弃了治疗。现在基本就是每天一颗盐酸西替利嗪，经济省事，还是医保药。最有意思的是曾经我去一家很有名的医院看寻麻疹，主治医生对这个病的了解还没有我多，弄的来看病的人还总问我！！呵呵！

回复
沈飞
2008年01月26日 18:08 编辑
我记得当时我有去看医生，有一个医生给了我一种药，但是药里含有激素。吃了这种药之后，荨麻疹状况真的没有犯过了，但是我整个人都肿起来了，所有的衣服，一个星期内都穿不上了。因为当时我有一个重要的面试，对体重的要求很严格，所以马上把药停了。但是我想，这个药的副作用虽然很大，可是对荨麻疹还真的是有帮助的。 不知道这个信息是否可以帮助你们。

回复
Qizhen
2007年12月19日 12:49 编辑
人工荨麻疹的英文是 factitious urticaria

回复
宇
2007年11月1日 13:19 编辑
1.       大概就是从高中二年级左右开始的吧!(1998年左右),  当时大病一场,吃乙旋螺旋霉索(可能是过了)引发的.

2.       最近越来越严重了(2007-11-1) 连眼睛都肿起来了,以前是不会上脸的.只是在脖子以下的皮肤表面出现
 

回复
julia
2007年10月2日 00:58 编辑
了解小儿荨麻疹的朋友吗？我两岁半的女儿得了此病，很难看好吗？知道的朋友请加我。
 

回复
lynn
2007年09月17日 08:46 编辑
额，我是今天5月份出现这种症状的，以为过几天就没事了。
可惜一直到现在还有，我也没去医院看，因为不想吃药，是药三分毒啊。
比较同意楼主的观点，充足的睡眠和运动比较重要。
我怀疑我是参加工作后还照样长时间玩游戏缺乏睡眠，长期不锻炼，长期吃食堂（单位食堂的饭菜营养可想而知）才得这个病的
而且我单位是造纸厂，洗澡用的水是厂内处理的，大量漂白粉
等我明年辞职，看看情况会不会有好转

回复
张薇
2007年08月6日 19:29 编辑
大家好，特别要感谢软先生的回复，可能是因为我以前身体太好了，连感冒都很少得，所以这次患病心里不免紧张，也许还因为年龄大了，怕死了，呵呵。看来这个病要陪伴我以后的日子了，很遗憾呀！看了大家的留言，我想我会多些锻炼的，尽量少吃药，是药三分毒，我要早日摆脱它。我也祝愿大家早日康复

回复
沈飞
2007年08月6日 16:27 编辑
我也是病人之一，咳，惨啊！不过奉劝那些还在寻找药方，想要治愈它的人，不要白费心机了。我的这个病已经11年了，吃了无数中药和西药，但都没有任何效果。到了今天，我早就放弃了。
还有，我觉得楼主说的有道理。我人在德国的时候，病情有减轻。这个月我刚回到国内，就发现病情有加重。看来空气可能真的多这个病有影响吧！ 

回复
Qizhen
2007年08月6日 09:14 编辑
我也只是病人，所以对这个病的了解就这么多了。我现在的人工荨麻疹虽然没有痊愈，但是也不怎么影响我的生活。所以我干脆不去管它了，就当没得过这病。

回复
张薇
2007年08月4日 21:14 编辑
 ruanqiahzn，你好，在网上看到你的关于人工荨麻疹的文章，我很想了解更多的有关这个病的知识，我是最近才发现得的这个病，心里也很紧张，吃不好，睡不好，下面是我的MSN地址，如果你能上网，请加我一下，我想了解等多的东西。jinn_830@hotmail.com

回复
右手戒指
2007年07月23日 15:36 编辑
看了大家的经历，是不是这种病应该少吃蘑菇啊？偏偏蘑菇菌类就是我的最爱，不知道何这个有没有关系？ 算了算我有这个病应该是15年整了。因为一直没太当回事，到现在估计治愈是没有希望了，成了慢性的了。不知道会不会遗传到下一代啊？

回复
右手戒指
2007年07月23日 15:29 编辑
我有这个症状应该是在小学四五年级，到现在已经有十几年的历史了。症状就是皮肤受到任何碰撞或者摩擦都会发红并突起。虽然不太影响正常生活，但是一般我在公众场合都不敢碰自己的脸，要不然会像个怪物，身体其它地方还可以有衣服遮盖。有时候好羡慕别人可以随便擦汗、揉眼睛啊！ 

回复
¤小鱼ｏО○
2007年07月11日 13:27 编辑
我以前都是好好的，不知道什么时候就开始有了。都好几个月了，真的好难受啊，得这个病得真的都是帅哥美女么？要真是的话得了也值了。现在我不 怎么吃药，总觉得不好，就是有的时候好难受，等我找到好方法再和大家说说好了。祝大家好运哦！

回复
¤小鱼ｏО○
2007年07月11日 13:27 编辑
我以前都是好好的，不知道什么时候就开始有了。都好几个月了，真的好难受啊，得这个病得真的都是帅哥美女么？要真是的话得了也值了。现在我不 怎么吃药，总觉得不好，就是有的时候好难受，等我找到好方法再和大家说说好了。祝大家好运哦！

回复
¤小鱼ｏО○
2007年07月11日 13:27 编辑
我以前都是好好的，不知道什么时候就开始有了。都好几个月了，真的好难受啊，得这个病得真的都是帅哥美女么？要真是的话得了也值了。现在我不 怎么吃药，总觉得不好，就是有的时候好难受，等我找到好方法再和大家说说好了。祝大家好运哦！

回复
安妮
2007年07月1日 10:38 编辑
帅哥美女们，我也你们的同伴，断续发了很多年了，很痛苦！今早找到一个不错的网站，给大家分享一下：
http://www.med66.com/web/zy_my_xmz/
中国人说偏方治大病，我就用很多偏方治好了亲朋好友们奇怪的病，呵呵，网站里很多中药无毒性或毒性很小的，希望大家跟我一样早日摆脱吖，加油耶

回复
刚
2007年06月15日 13:15 编辑
有荨麻疹真是件麻烦事。我可怜的老婆一到夏天，总会有些过敏。北京的天气和环境又是如此糟糕，唉。不过楼主说的帅哥美女得荨麻疹我是绝对赞同，所以我没得。

回复
晓婷
2007年06月3日 22:50 编辑
我看大家的留言，可我好像觉得没有一个看好的，怎么办，难道这种病就看不好了吗？我现在看中医，给我开了中药让我服用，开始我的信心很高，怎么多年我的病终于可以找到治疗的办法了，因为我一直没好，可我觉得没有效果，这里面有没有看好的，大家给我方子，我真的太痛苦了！！！！

回复
菲
2007年06月2日 09:08 编辑
也不知道自己什么时候有这个症状的啦，反正很小，我自己还在自己胳膊上写字那，我以为就我一个人有这个病，我以为我血的问题，我也没敢和大人说，现在我21了，都N年了，我也没在意，就是碰到了划到的会红红的一条一条的，，我什么都吃，没有忌口的，病情也没在严重，就这么过吧，也不影响什么，不疼不痒的，能和病友交流交流我很开心，哈哈

回复
无脚的鸟
2007年05月11日 21:05 编辑
找到这么多知音，真是太让我激动了，快有四年了，这几年来我一直都饱受人工荨麻疹的痛苦，到现在一点好转的迹象都没有，不过我一直都没有怎么忌口，目前唯一总结出来的有效减轻办法就是：规律的作息，避免生气，勤洗澡。另外对于我们女生来说，每月那几天之前好像反应尤其重，今天看了大家的帖子后，我下定决心，从现在开始忌口，远离我最爱的辣椒、大虾、蘑菇！！！！！！我们大家一起努力，争取早日摆脱这个痛苦，做回帅哥美女！！！！

回复
zheng
2007年05月9日 01:00 编辑
得人工荨麻疹的都是帅哥美女，哈哈哈哈，同意。
 

回复
eiko
2007年03月30日 06:54 编辑
今天早上起得早，没事就上网想看看有关荨麻疹的症状和治愈方法。前一段时间回国检查身体，问了一下皮肤科得知自己的症状是人工荨麻疹！哎呀，，听说还是不好治，一般痊愈不了呢！！这病以前从没听过，我一直身体也不错的，总觉得是不是误诊啊，万一是别的呢，现在看来，这种病慢慢普遍起来了。同病相怜的朋友只有一起努力锻炼身体好了。幸好看了帖子，帮了大忙耶！

回复
Rach
2007年02月10日 16:04 编辑
原来不是只有我一个有这种病的!这个病真是令人好难受呀!不过,我现在都在看中医,情况暂时得都控制,就是不知道什么时候才回好呀,痛苦呀,痛苦!

回复
vivian
2007年02月6日 20:41 编辑
我也是这样，都快两个月了，吃了防风通圣丸，不见什么效果，难受~~

回复
云
2006年12月9日 10:20 编辑
荨麻疹我是去年年底得的，莫名其妙就得了。刚开始没注意好好治疗，现在转为慢性了，很痛苦啊。吃了西药中药，停药就复发了。

回复
Unknown
2006年12月1日 19:16 编辑
我应该是属于那种顽固慢性荨麻疹，因为我没有特定发作的条件，也就是说今天我冷了就发了，明天就不发；或者今天饿了又发了；明天同样情况不发；等等等等。我总觉得是和阿姨有关，2000年冬天前，吃妈妈的饭从来也没发过，就是那年我换了地方，那个阿姨老是烧蘑菇炒这个蘑菇炒那个，因为她会做得东西太少了，而且她觉得蘑菇很好。只吃了一个月，我就劳了个荨麻疹。后来也有相关文章说这些菌菇类的食物食太多会诱发过敏体质爆发，我真是非常的后悔，本来我只是有点隐性的过敏体质，就是晕车而已。
 
所以，同志们一定要注意休息和排毒，另外就是补充缺乏的东西，比如钙、维生素C和B6，女同胞要尽量不吃冰冷的食物，否则会加重寒毒在体内的淤积。
 
我的一些观点都是自己研究来的，现在这些医生不把人看死就不错了。

回复
Unknown
2006年12月1日 19:04 编辑
你好。
今天无意当中看到你关于荨麻疹的抱怨，真是感同身受。
 
我在2000年的冬天，一个晚上突然腿部很痒，然后一直痒上来，痒到头部，足足痒了2个小时才停止。由于我身体一直很好，并且上海的医院都差的一塌糊涂，所以我也没在意，就直接到楼下药方问了一声，然后他们告诉我是荨麻疹，也就是“风疹块”。这下我想起来了，我妈妈也有这个毛病，她是过敏体质，生孩子的时候劳下的毛病，原来是遗传的。当时，他们推荐我买了西可韦盐酸西替利嗪。
 
刚开始，我几乎是药一放到嘴里就没那么痒了。所以直到两年以后，一天不止发一次而是两次三次的时候，我才开始关注这个毛病。因为我痒起来时间实在是太长了，而且随时随地都可能发作，每一寸肌肤都可能痒到不抓不行。
 
后来的一年，我几乎是看遍了上海的大小医院，专家门诊，每个都是1分钟就判你死刑。我也试过私人医院，花了很多钱做了各种各样的测试，吃了各种各样的中药和西药……现在，我还是自己在买盐酸西替利嗪吃。因为医院开给我别的药，要么没有效果，要么就是让我睡得死死的，天塌下来也不会知道。
 
不过有一点我承认，的确是要好好休息，如果我睡觉睡的很好，也尽量不吃海鲜、羊肉、辛辣食物和饮酒，我基本上是两天发作一次。
 
最近我想尝试一下偏方
1. 用十余棵香莱的根须洗净切段，煮5分钟，调上蜂蜜，连吃带饮。连续用3日，每日1剂
*《血液的革命》里说香菜是可以清理血液的，所以此方法铁定有效
2.取白酒、食醋以1:2混合成液，用此药涂搽患处，数分钟后即可见效
3.经过临床验证还是拔罐配合后背俞穴的放血拔罐效果最为好

回复
濛
2006年10月12日 10:40 编辑
我是自从搬到了新装修办公室就开始背上痒，后来过年离开办公室一段时间还是不停的再犯，后来去看说我是人工荨麻疹，吃开瑞坦，说开始每天一片，然后控制下来以后就每两天一片，然后稳定下来以后再慢慢减下来，可是我开始一天一片吃了一个多星期以后改成2天一片发现还是不行，不知道这个药要吃多久才行，郁闷致死。

回复
斯
2006年09月27日 12:46 编辑
还有推论2:不少还是火星来的…

回复
sharon
2006年09月26日 21:49 编辑
这个论调很好玩,患此病者帅哥美女多.
听说利嗪片和乌蛇止痒不错,可以试试.

回复
yan
2006年09月20日 20:38 编辑
我也是有这个病，好好的，突然有一天就开始痒了。不过我没有吃药，我想你说的很有道理，加强体质最重要！！！

回复
PPYu
2006年08月1日 17:30 编辑
我也被这个顽固的皮肤病折腾了很久 虽然不是每晚都很痒 但是晒过太阳都会很难受
这个真的可能与每个人的皮肤特性有关
 
楼下的有一位提到的摸的药叫:炉甘石洗剂 倒是很便宜 我在医院开的 不妨试试 但是我一般上半身痒痒 所以总不能把上半身全抹上吧 哎 反正挺苦恼的
 
后来我发现洗澡后 会好点 所以要多清洁卫生
 
现在在中医医院看专家 吃中药 但是说实话那个烂专家都没有给我保证什么时候能好!
 
还是希望有共同遭遇的朋友们早日康复吧!!

回复
提卡
2006年06月25日 12:22 编辑
呵呵 原来有这么多同道中人啊 我是原来就容易过敏 不过还没出现人工荨麻疹 去年秋天来的沈阳 今年春天出了水痘之后人工荨麻疹也接着出现了 我也在服用盐酸西替利嗪 不过好像作用不明显

回复
s
2006年06月1日 08:48 编辑
亲..我也是这个毛病.去年从HK回来上海的那个春天之后,就开始莫名其妙的得了荨麻疹.还有慢性风疹,虽然都不传染.但是实在是难熬!现在拖了有一年半的时间了.医院说也是人工型的荨麻疹,是不能根治的…在云南的时候本想尝试下彝医….但是几副中药效果肯定是甚微的.
盐酸西替利嗪,这个药我钢开始也吃的.一开始,不吃就受不了…米办法断药,晚上睡不着.后来…忍了一段时间.基本摆脱药物控制,但是,每天都会过敏.很困扰!
郁闷..彝医说我这是什么内湿,血热..MOROSE….

回复
Yan
2006年05月13日 23:50 编辑
 
 
  唉。。。原来咱们是知音。。。。
  我这个病好多年了，是遗传我妈妈的。
  医生说没法痊愈~~~
 
 
 

回复
大用
2006年02月7日 13:55 编辑
我儿子最近也得了这个病（才1岁8个月），好麻烦呀？听说只能吃中药治疗，各位有什么高见没有？？

回复
Unknown
2005年12月3日 22:07 编辑
唉!我从来都认为我身体搔痒是正常的,到后来我朋友身上也出现了这种情况,我才担心是不是被我传染,于是就跑到医院一看,原来发现是得病了!真是郁闷!不过后来又一询问,知道这个病是不传染得!心里又一阵平静!但是,想想就觉得冤,因为本人的父亲是一个医生,但是我却从来都不觉的怎么样,有病了也不问!白白浪费了n多年的时间!要不,也许这个病早就治好了呢!今天知道这个病叫什么名字,一上网查,发现和自己的症状非常吻合!晕死!然后发现,自己可能是得得比较早得,因为已经有10多年的时间!发现自己是多么的不知道珍惜自己的身体!现在深深的忏悔,并下定决心向楼主学习!

回复
Iris
2005年11月7日 23:05 编辑
一种药最好不要长期服用，扑尔敏也是。

回复
Iris
2005年11月7日 23:04 编辑
在深圳的时候，每一年都会发病一次，一般都是春夏季节，痒的受不了。坚持抹炉甘石比较有效，有个朋友也是在美国，过敏后看了很多医生都没效，最后来中国用炉甘石治好的，虽然是很便宜的药品。我还去医院买了点药，可惜不记得名字了。在智利还好，目前还没有发病。最近一直有坚持做瑜珈。

回复
旦阳
2005年09月24日 14:44 编辑
是啊，体质好了，抵抗力强了，应该就不容易过敏了。祝早日完全康复。◕‿◕

回复
yumi
2005年09月23日 12:55 编辑
偶来啦~~呵呵

回复
水映天涯
2005年09月22日 14:49 编辑
哇你生病了~~~好些了吗?
